src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -6017,10 +6017,14 @@ static int findPartGroupNumber(int* partGroupEnd)
 //  scoreInstrument
 //---------------------------------------------------------
 
-static void scoreInstrument(XmlWriter& xml, const int partNr, const int instrNr, const QString& instrName)
+static void scoreInstrument(XmlWriter& xml, const int partNr, const int instrNr, const QString& instrName,
+                            const Instrument* instr = nullptr)
 {
     xml.startElementRaw(QString("score-instrument %1").arg(instrId(partNr, instrNr)));
     xml.tag("instrument-name", instrName);
+    if (instr && !instr->instrumentId().isEmpty() && !MScore::testMode) {
+        xml.tag("instrument-sound", instr->instrumentId());
```

Comment:
Sorry, one more thing: `instrumentId()` has been renamed to `musicXmlId()` in the master branch.

Reply:
Oops, needs to wait until tomorrow then

Reply:
Done (hopefully)